### DISCUSSION TITLE
Get installed version of GitExtensions from command line

### DISCUSSION THREAD

**Discussion by Opligam** [gitextensions/gitextensions#9783] [General] [answered]
Hi

Is there a way to check what is the installed version of GitExtensions from a command line or by checking the registry ?

**Comment by gerhardol** [comment, marked as answer]
You can read the version from the .exe. In PowerShell:
`(get-command ${Env:Programfiles(x86)}\GitExtensions\GitExtensions.exe).FileVersionInfo.ProductVersion`

**Reply by Opligam** [reply]
Checking .exe file version is good enough for me. thanks.

**Comment by RussKie** [comment]
I don't think so. Could you please elaborate on the use case?
